Automattic/wp-calypso
# Jetpack Backups: Cleanup any product upsells

We decided to remove any upsells from the v1 of Jetpack Backups UI in Calypso. This PR removes any upsell logic that we already had created.

As a next step, we should make all states aware of the current plan / product.

#### Changes proposed in this Pull Request

* Jetpack Backups: Cleanup any product upsells

#### Testing instructions

* Checkout this branch.
* Go to http://calypso.localhost:3000/plans/yearly/:site where `:site` is a Jetpack site.
* Verify that regardless of what Jetpack Backup product or plan you have on your side, you don't get any upsell buttons.
* Verify that when you don't have a plan/product, everything still works well and you can buy a product without any problems.
